src/ckanext-ann-arbor/ckanext/ann_arbor/views/dataset.py

```
@@ -0,0 +1,126 @@
+
+dataset = Blueprint(
+    "ann_arbor_dataset",
+    __name__,
+    url_prefix="/dataset",
+    url_defaults={"package_type": "dataset"},
+)
+
+
+class CreateView(BaseCreateView):
+
+    def post(self, package_type):
```

Pylint (3.3.7)

[refactor] Too many return statements (10/6)

(R0911)